python/nano/test/onnx/tf/test_inc_onnx.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from unittest import TestCase
+import tensorflow as tf
+from tensorflow.keras.applications.resnet50 import ResNet50
+import numpy as np
+from bigdl.nano.tf.keras import Model
+
+
+class TestONNX(TestCase):
+    def test_model_trace_onnx(self):
+        model = ResNet50(weights=None, input_shape=[224, 224, 3], classes=10)
+        model = Model(inputs=model.inputs, outputs=model.outputs)
+        input_examples = np.random.random((100, 224, 224, 3))
+        input_features = np.random.randint(0, 10, size=100)
+        
+        train_dataset = tf.data.Dataset.from_tensor_slices((input_examples,
+                                                            input_features)).batch(1)
```

Comment:
Here it seems calibration dataset for onnx+int8 requires `batch(1)`.
Does int8 quantization has this limit?
Maybe give more details for specific dataset requirements in the annotation of `calib_dataset: tf.data.Dataset`.

Other choice is do `batch(1)` within quantization.py and `unbatch` at last to align with int8.

Reply:
> Other choice is do batch(1) within quantization.py and unbatch at last to align with int8.

good idea, take it.